wedding_site/static/templates/layout.html

```diff
@@ -39,8 +39,7 @@
 							<ul class="nav navbar-nav menu_nav ml-auto">
 								<li class="nav-item"><a class="nav-link" href="/">Home</a></li> 
 						<!-- 		<li class="nav-item"><a class="nav-link" href="rsvp.html">RSVP  </a></li> -->
-								<li class="nav-item"><a class="nav-link" href="/accomodation">Accomodation</a></li>
-								<li class="nav-item"><a class="nav-link" href="/venue">Venue</a></li>
+								<li class="nav-item"><a class="nav-link" href="/venue">Accomodation and Venue</a></li>
```

Comment:
Id like to change this back, but we can do that at a later time. Having these separated makes sense to me.